packages/dialtone-vue2/components/popover/popover.vue

```diff
@@ -1012,41 +981,13 @@ export default {
       });
     },
 
-    //  ============================================================================
-    //  $ HOVERCARD
-    //  ----------------------------------------------------------------------------
-
-    setInTimer () {
-      this.inTimer = setTimeout(() => {
-        this.isOpen = true;
-      }, this.enterDelay);
+    onMouseEnter () {
+      this.$emit('mouseenter-popover');
```

Comment:
Should the event be called "mouseenter-popover-anchor" since there could also be the case of the mouse entering the popover dialog?